src/helpers/apiCalls.tsx

```diff
@@ -42,6 +43,20 @@ export const getAirQualityData = (lat: number, long: number) => {
     .catch((error) => console.error(error))
 }
 
+export const getHomeData = async (lat: number, long: number) => {
```

Comment:
Looks great, If you both aren't using getAllData I am probably gonna change that a bit. Great work you two.

Reply:
We aren't for now! It's just hooked up to Results/Search right now they way it is, so if any changes are made that may need to be adjusted.